metadata.yaml

```diff
@@ -15,6 +15,10 @@ provides:
     interface: k8s-service
     schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/k8s-service.yaml
     versions: [v1]
+  monitoring:
```

Comment:
We call it `metrics-endpoint`: monitoring is more than just collecting metrics ;-)